src/components/backend-ai-session-view.ts

```diff
@@ -444,7 +444,7 @@ export default class BackendAiSessionView extends BackendAIPage {
     return html`
       <lablup-activity-panel title="${_t('summary.ResourceStatistics')}" elevation="1" autowidth>
         <div slot="message">
-          <backend-ai-resource-monitor location="session" id="resource-monitor" ?active="${this.active === true}"></backend-ai-resource-monitor>
+          <backend-ai-resource-monitor location="session" id="resource-monitor" ?active="${this.active === true}" direction="vertical"></backend-ai-resource-monitor>
```

Comment:
Please explain why you cast the 'direction' on resource monitor.

Reply:
The initial code was written to show "scaling-group-select-box" in the resource monitor only if the direction is vertical.
So I also set direction to vertical to show the select box on the session page.
However, since the select box is shown everywhere using the resource monitor, I modified it to show the select box by default, not by direction.

Reply:
Also, unlike other pages (summary, import page), the session page displays the select box horizontally. So I changed the following items.
1. Added css code when direction is horizontal.
2. Change the updateScalingGroupSelector() to operate regardless of direction so that the scaling group can be updated even when the direction is horizontal (Previously, it only works when direction is vertical)